src/services/ImageRepository.cpp

```diff
@@ -353,7 +353,10 @@ HBITMAP ImageRepository::LoadLocalPNG(const std::string& sFilename, size_t nWidt
     // Scale the original IWICBitmapSource to the client rect size and convert the pixel format
     CComPtr<IWICBitmapSource> pToRenderBitmapSource;
     if (SUCCEEDED(hr))
-        hr = ConvertBitmapSource({ 0, 0, static_cast<LONG>(nWidth), static_cast<LONG>(nHeight) }, pOriginalBitmapSource, *&pToRenderBitmapSource);
+    {
+        RECT myRect{ 0, 0, to_signed(nWidth), to_signed(nHeight) };
+        hr = ConvertBitmapSource(myRect, pOriginalBitmapSource, *&pToRenderBitmapSource);
```

Comment:
I was looking over other files that may have been related to this before. I've seen a lot of usages as rvalues (temporaries) as lvalues, this was one such case. The other one's weren't too serious so I didn't change them. If you want to use as an rvalue, consider appending && to the typename.